src/excercise2.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Created by ttn on 28/12/16.
+ */
+
+/**
+ * Add ability to above component to change it's state ( temperature ) every 5 seconds with some random value between -10 to 110.
+ *
+ */
+
+
+import React from 'react'
+
+export default class Water extends React.Component{
+
+  constructor(props){
+    super(props)
+    this.state={
+      temperature : 0
+    }
+  }
+
+  componentDidMount(){
+    this.timerId = setInterval(
```

Comment:
clearInterval during unmount phase to avoid memory leakage when component is unmounted